src/interactive.test.js

```diff
@@ -0,0 +1,87 @@
+const jsdom = require('jsdom');
+
+const checkCompleted = require('./interactive.js');
+
+const { JSDOM } = jsdom;
+
+const dom = new JSDOM(`<!DOCTYPE html><ul class="d-flex todo-lists"></ul>`); // eslint-disable-line
+const document = dom.window.document; // eslint-disable-line
+const window = dom.window.document; // eslint-disable-line
+
+const localStorageMock = (() => {
+  let store = {};
+
+  return {
+    getItem(key) {
+      return store[key];
+    },
+
+    setItem(key, value) {
+      store[key] = value;
+    },
+
+    clear() {
+      store = {};
+    },
+
+    removeItem(key) {
+      delete store[key];
+    },
+
+    getAll() {
+      console.log(store);
+    },
+  };
+})();
+
+Object.defineProperty(window, 'localStorage', { value: localStorageMock });
+
+describe(' checking status completed localstorage', () => {
+  document.body.innerHTML = `
+        <li class="list-container d-flex">
+          <input type="checkbox" class="checkbox" checked>
+          <p class="items show"   contenteditable="true">Hello Todo's</p>
+          <i class="fas fa-ellipsis-v  dots"></i>
+          <i class="far fa-trash-alt show delete"></i>
+        </li>
+        <button class="btn">Clear all Completed</button>
+        `;
+  test('function to check the complete statues false', () => {
+    const list = localStorageMock;
+    // addTodoToList('take a break', false, 0, list);
+    const task = document.querySelector('.items').innerText;
+    const element = document.querySelector('.checkbox');
+    const bool = false;
+    checkCompleted(list, task, element, bool);
```

Comment:
- [ ] One test case is failing. Kindly ensure to improve upon this aspect.

**Failing test case:**
![image](https://user-images.githubusercontent.com/12745474/141347378-adb5f66f-9c04-4bb3-887e-f70b84e349c0.png)

Reply:
Fixed the issue for the completed function. Thank you.